aws/vpc/README.md

```diff
@@ -4,46 +4,50 @@ be useful.
 
 ## What this does
 
- - Create VPC named after `app_name` and `app_env`
- - Create public and private subnets for each `aws_zones` specified
- - Provision a Internet Gateway and configure public subnets to route through it
- - Provision a NAT Gateway (or use an existing Transit Gateway) and configure private subnets to route through it
- - Create a DB subnet group including all private subnets
+- Create VPC named after `app_name` and `app_env`
+- Create public and private subnets for each `aws_zones` specified
+- Provision a Internet Gateway and configure public subnets to route through it
+- Provision a NAT Gateway (or use an existing Transit Gateway) and configure private subnets to route through it
+- Create a DB subnet group including all private subnets
+- Optionally allocate IPv6 CIDR blocks, egress-only internet gateway, and default IPv6 routes
 
 ## Required Inputs
 
- - `app_name` - Name of application, ex: Doorman, IdP, etc.
- - `app_env` - Name of environment, ex: prod, test, etc.
- - `aws_zones` - A list of zones to create subnets in, ex: `["us-east-1c", "us-east-1d", "us-east-1e"]`
+- `app_name` - Name of application, ex: Doorman, IdP, etc.
+- `app_env` - Name of environment, ex: prod, test, etc.
 
 ## Optional Inputs
 
- - `enable_dns_hostnames` - default `false`
- - `create_nat_gateway` - default `true`
- - `private_subnet_cidr_blocks`
- - `public_subnet_cidr_blocks`
- - `vpc_cidr_block`
- - `use_transit_gateway` - default `false`
- - `transit_gateway_id` - required if `use_transit_gateway` is true
- - `transit_gateway_default_route_table_association` - default `true`
- - `transit_gateway_default_route_table_propagation` - default `true`
+- `aws_zones` - A list of zones to create subnets (max 8) - default: `["us-east-1c", "us-east-1d", "us-east-1e"]`
+- `enable_dns_hostnames` - default `false`
+- `create_nat_gateway` - default `true`
+- `private_subnet_cidr_blocks` - default: `["10.0.11.0/24", "10.0.22.0/24", "10.0.33.0/24", "10.0.44.0/24"]`
+- `public_subnet_cidr_blocks` - default: `["10.0.10.0/24", "10.0.20.0/24", "10.0.30.0/24", "10.0.40.0/24"]`
+- `vpc_cidr_block` - default: `"10.0.0.0/16"`
+- `use_transit_gateway` - default `false`
+- `transit_gateway_id` - required if `use_transit_gateway` is true
+- `transit_gateway_default_route_table_association` - default `true`
+- `transit_gateway_default_route_table_propagation` - default `true`
+- `ipv6_enable` - default `false` 
```

Comment:
We have `enable_dns_hostnames`, so it might be better to use `enable_ipv6` instead.